functorch/functorch/_src/eager_transforms.py

```diff
@@ -785,11 +785,20 @@ def jvp(func: Callable, primals: Any, tangents: Any, *, strict: bool = False, ha
          >>> assert torch.allclose(output, x + y)
 
     """
+
+    return _jvp_with_argnums(func, primals, tangents, argnums=None, strict=strict, has_aux=has_aux)
+
+
+# only to be used with jacrev to allow for all passed variables to be wrapped correctly, even if they don't participate
+# in autograd
+def _jvp_with_argnums(func: Callable, primals: Any, tangents: Any, argnums: Optional[argnums_t], *,
```

Comment:
I'd probably add a comment that the error messages are written from the perspective of the `jvp` API, not the `_jvp_with_argnums` API... Otherwise it gets a bit confusing reading the code

Reply:
Added!